manifests/init.pp

```diff
@@ -166,6 +171,7 @@
   Boolean $mime_types_preserve_defaults                      = false,
   Optional[String] $repo_release                             = undef,
   $passenger_package_ensure                                  = 'present',
+  String $passenger_package_name                             = $nginx::params::passenger_package_name,
```

Comment:
Same here. Please document the parameter and enforce a minimal string length of 1:
```suggestion
  String[1] $passenger_package_name                          = $nginx::params::passenger_package_name,
```

Reply:
Sorry for the delay, I just pushed a fix!